sdk/eventhub/azure-eventhubs/azure/eventhub/client.py

```diff
@@ -47,6 +48,19 @@ class EventHubClient(EventHubClientAbstract):
 
     """
 
+    def __init__(self, host, event_hub_path, credential, **kwargs):
+        super(EventHubClient, self).__init__(host, event_hub_path, credential, **kwargs)
+        self._conn_manager = get_connection_manager(**kwargs)
+
+    def __del__(self):
+        self._conn_manager.close_connection()
```

Comment:
Is there a reason we're defining `__del__`? I don't think we do for any of our other clients...

Reply:
This is to guard if a user forgets to close a client (connection). 
There is no async __del__, so the async client doesn't have this.
I have no problem if we remove __del__ for sync